src/test/scala/org/broadinstitute/dsde/workbench/sam/service/UserServiceSpecs/CreateUserSpec.scala

```diff
@@ -27,71 +20,42 @@ class CreateUserSpec extends UserServiceTestTraits {
   val allUsersGroup: BasicWorkbenchGroup = BasicWorkbenchGroup(CloudExtensions.allUsersGroupName, Set(), WorkbenchEmail("all_users@fake.com"))
   val blockedDomain = "blocked.domain.com"
 
-  describe("UserService.createUser") {
-    describe("returns an enabled UserStatus that has NOT accepted ToS") {
-      it("when user has an AzureB2CId and a GoogleSubjectId") {
-        // Setup
-        val userWithBothIds = genWorkbenchUserBoth.sample.get
-        val userService = TestUserServiceBuilder()
-          .withAllUsersGroup(allUsersGroup)
-          .build
-
-        // Act
-        val userStatus = runAndWait(userService.createUser(userWithBothIds, samRequestContext))
-
-        // Assert
-        inside(userStatus) { status =>
-          status should beForUser(userWithBothIds)
-          "google" should beEnabledIn(status)
-          "ldap" should beEnabledIn(status)
-          "allUsersGroup" should beEnabledIn(status)
-          "adminEnabled" should beEnabledIn(status)
-          "tosAccepted" shouldNot beEnabledIn(status)
-        }
-      }
-
-      it("when user has an AzureB2CId but no GoogleSubjectId") {
-        // Setup
-        val userWithOnlyB2CId = genWorkbenchUserAzure.sample.get
-        val userService = TestUserServiceBuilder()
-          .withAllUsersGroup(allUsersGroup)
-          .build
-
-        // Act
-        val userStatus = runAndWait(userService.createUser(userWithOnlyB2CId, samRequestContext))
-
-        // Assert
-        inside(userStatus) { status =>
-          status should beForUser(userWithOnlyB2CId)
-          "google" should beEnabledIn(status)
-          "ldap" should beEnabledIn(status)
-          "allUsersGroup" should beEnabledIn(status)
-          "adminEnabled" should beEnabledIn(status)
-          "tosAccepted" shouldNot beEnabledIn(status)
-        }
-      }
-
-      it("when user has GoogleSubjectId but no AzureB2CId") {
-        // Setup
-        val userWithOnlyGoogleId = genWorkbenchUserGoogle.sample.get
-        val userService = TestUserServiceBuilder()
-          .withAllUsersGroup(allUsersGroup)
-          .build
-
-        // Act
-        val userStatus = runAndWait(userService.createUser(userWithOnlyGoogleId, samRequestContext))
-
-        // Assert
-        inside(userStatus) { status =>
-          status should beForUser(userWithOnlyGoogleId)
-          "google" should beEnabledIn(status)
-          "ldap" should beEnabledIn(status)
-          "allUsersGroup" should beEnabledIn(status)
-          "adminEnabled" should beEnabledIn(status)
-          "tosAccepted" shouldNot beEnabledIn(status)
+  val userWithBoth = (genWorkbenchUserBoth.sample.get, "user with google and azure ids")
```

Comment:
I like the tuples, my only concern is that it makes it a little harder to read it in the code - although the test output is still clear. Maybe we could add a comment or a readme somewhere suggesting that as an easy way to debug/check expected behavior?  Maybe everyone would think to do that, but I'm not used to the test results printing out in such a helpful manner so I'm not sure if it would occur to me if I were a dev looking at this for the first time.